server/src/app/views.py

```diff
@@ -125,7 +129,9 @@ def authorizationCallback():
 """
 @app.route('/api/data/benefit',methods=['GET'])
 def getPatientEOB():
-    if (loggedInUser.get('eobData') == ''):
-        return ''
+    if (loggedInUser != None 
+        and loggedInUser.get('eobData') != None
+        and loggedInUser.get('eobData') != ''):
+        return json.loads(loggedInUser.get('eobData'))
     else:
-        return json.loads(loggedInUser.get('eobData'))
\ No newline at end of file
+        return ''
```

Comment:
Would it make sense to return an empty object here instead?

Reply:
I was doing that before, but then the parsing on the front-end failed straight out.  We should, in the next iteration, tackle how to handle not returning any results on the front-end.